safe-global/safe-wallet-web
# Feat swaps fees [SW-34]

## What it solves
Swaps now support dynamic fees.

## How to test it
Navigate to the swaps widget - enter different amounts to sell. The applied fee should depend on the fiat amount swapped.

## Screenshots

## Checklist
